app/views/layouts/_nav.html.erb

```diff
@@ -46,7 +49,7 @@
             <li><%= scp_link_to "<span class='fas fa-tags fa-fw'></span> My Annotations".html_safe, user_annotations_path, class: 'check-upload', id: 'my-annotations' %></li>
             <li><%= scp_link_to "<span class='fas fa-book fa-fw'></span> My Studies".html_safe, studies_path, class: 'check-upload', id: 'my-studies' %>
             <li><%= scp_link_to "<span class='fas fa-dollar-sign fa-fw'></span> My Billing Projects".html_safe, billing_projects_path, class: 'check-upload', id: 'my-firecloud-projects' %>
-						<li><%= scp_link_to "<span class='fas fa-plus fa-fw'></span> Add a Study".html_safe, new_study_path, class: 'check-upload' %></li>
+						<li><%= scp_link_to "<span class='fas fa-plus fa-fw'></span> Create a Study".html_safe, new_study_path, class: 'check-upload' %></li>
```

Comment:
I like keeping the old link.  To me, that menu serves as a 'list of everything you can do', it makes sense to list everything, even items that may be also be listed elsewhere.  Also, since the new link is an experiment that might get reverted/altered, I'd rather not change existing user workflows until it stabilizes

Reply:
Reasonable, especially since designer agrees!